package.json

```diff
@@ -72,14 +72,12 @@
     "@types/lodash": "^4.14.202",
     "@types/node": "^20.0.0",
     "@types/offscreencanvas": "^2019.7.0",
-    "@typescript-eslint/eslint-plugin": "^7.3.1",
-    "@typescript-eslint/parser": "^7.3.1",
     "ace-builds": "^1.4.12",
     "coveralls": "^3.1.0",
     "escodegen": "^2.0.0",
     "eslint": "^8.57.0",
```

Comment:
Are we not bumping to v9 together with flat config?

Reply:
Though since technically flat config is supported since v8.something, it's not necessary

Reply:
For that exact reason I left it alone

Reply:
Sounds good!